modules/proc_configuring-notificationsconfiguration-crd-by-using-the-ocp-cli.adoc

```diff
@@ -0,0 +1,54 @@
+:_mod-docs-content-type: PROCEDURE
+[id="configuring-notificationsconfiguration-crd-by-using-the-ocp-cli_{context}"]
+= Configuring NotificationsConfiguration (CR) by using the {OCP} CLI
+
+You can configure NotificationsConfiguration (CR) by using the {OCP} CLI.
+
+.Prerequisites
+
+* You have access to an {OCP} cluster with `cluster-admin` privileges.
+* You have installed the {gitops-title} Operator on your cluster.
+* You have enabled notifications for the Argo CD instance.
+
+.Procedure
+
+. Log in to the {OCP} cluster.
+
+. Run the following command to enable notifications:
++
+[source,yaml]
+----
+oc edit argocd openshift-gitops -n openshift-gitops
+----
+
```

Comment:
Adding an example output would be nice.
```suggestion
.Example output
[source,terminal]
----
$ output
----
```

Reply:
I do not remember what the output was so for the moment I am not including this feedback.

Reply:
Fixed.